CHANGES_NEXT_RELEASE

```diff
@@ -23,6 +23,7 @@
 - Fix: invalid regex patterns detection (#968)
 - Fix: Better error returned on invalid geoquery (Issue #2174)
 - Fix: Better error returned on attribute not found in GET /v2/entities/<entity-id>/attrs/<attr-name>/value (Issue #2220)
+- Fix: Distinguish between strings and numbers in q string filter (Issue #1129). NOTE: we don't know exactly *when* this issue was fixed ...
```

Comment:
I remember we did a big refactor of the "q stuff" not so much time ago, moving the parsing logic from monoBackend functions to a new class for representing filters (not fully sure...).  Maybe this was fixed with that refactoring?